web/src/components/Trace/TraceComponent/ResizableDrawer.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
```

Comment:
I think this should be at the `/components` level as it seems to be generic enough that we can use on different instances outside of the trace